app/controllers/application_controller.rb

```diff
@@ -1,3 +1,29 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
+
+  before_action :require_login
```

Comment:
This should not be present here, you should focus on requiring login at action/controller level until lesson 6 where you learn about how to do this properly - via actors.